src/utils/error.js

```diff
@@ -1,18 +1,9 @@
+
 /**
- * Class representing errorHandlers.
- * @extends Error
- */
-class ErrorHandler extends Error {
```

Comment:
Why remove this?

Reply:
wow, didn't notice I removed. Brought it back now